woocommerce/woocommerce-android
# [Stats] Use summary stats endpoint to fetch total visitor count

Closes: #11171 

### Description
This PR uses https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2983 to use the summary stats endpoint to fetch the visitor's total count; the count is fetched only for predefined ranges.

In this PR, I also made changes to the granularity we use for `2 days` ranges, please check p1711724047631339-slack-C03L1NF1EA3 for more information.

### Testing instructions
I don't have any store that has enough visitor stats to showcase the difference between the `views` and `visitors`, but I think that's OK for now.

1. Open the app.
2. Test the predefined ranges, and confirm total visitors is shown.
3. Optionally: compare the visitor stats count with what's shown in Web and iOS.
4. Create a custom range with a single day.
5. Confirm the visitor's count is shown.
6. Create a custom range with 2 or more days.
7. Confirm the visitor's count is hidden.
8. Tap on a graph point.
9. Confirm the visitor's count is shown.

### Images/gif
